debian/libibverbs-dev.links

```diff
@@ -0,0 +1,16 @@
+usr/share/man/man3/ibv_alloc_pd.3		usr/share/man/man3/ibv_dealloc_pd.3
```

Comment:
Yes, you are right. My bad. I checked that all symlinks are specified in libibverbs/man/CMakeLists. Thus I can drop that commit.